joommarketlink.php

```diff
@@ -45,6 +45,8 @@ public function __construct(&$subject, $config)
                                                         'link'  => 'index.php?option=com_virtuemart&view=productdetails&virtuemart_product_id=%s'),
                               'hikashop'      => array( 'icon'  => 'media/com_hikashop/images/icons/icon-16-hikashop.png',
                                                         'link'  => 'index.php?option=com_hikashop&view=product&layout=show&product_id=%s'),
+                              'joomshopping'  => array( 'icon'  => 'media/joomgallery/images/basket.png',
+                                                        'link'  => 'index.php?option=com_jshopping&controller=cart&task=add&category_id=1&product_id=%s'),
```

Comment:
Die category_id ist hier jetzt mit 1 hart kodiert. Funktioniert das auch, wenn die Produkte in anderen Kategorien liegen?